docs/example.rst

```diff
@@ -19,39 +16,33 @@ Example
       -[channel] *120*456#
 
 
-   Favourite Colour
-   ~~~~~~~~~~~~~~~~
+   ### Favourite Colour
```

Comment:
On second thought, I find https://github.com/praekelt/numl/blob/feature/issue-4-design-fixes-from-user-testing/docs/example.rst almost illegible. Are we sure about this change?